data-integration/admin_units.py

```diff
@@ -1,41 +1,38 @@
 #!/usr/bin/env python3
-# -*- coding: utf-8 -*-
 """
-Created on Thu Oct 22 17:53:54 2020
+Obtain GISCO admin units that will be loaded as ds_id = 0.
+No update check is performed here.
 
 @author: giuseppeperonato
 """
 
 import logging
 import os
+import sys
 
 import geopandas as gpd
 import pandas as pd
 import utilities
 from pyproj import CRS
 
-# Constants
 # GISCO datasets GEOJSON EPSG:4326 1:1milion
 GISCO_DATASETS = {
     "countries": "https://gisco-services.ec.europa.eu/distribution/v2/countries/geojson/CNTR_RG_01M_2020_{}.geojson",
     "nuts": "https://gisco-services.ec.europa.eu/distribution/v2/nuts/geojson/NUTS_RG_01M_2021_{}.geojson",
     "lau": "https://gisco-services.ec.europa.eu/distribution/v2/lau/geojson/LAU_RG_01M_2019_{}.geojson",
 }
+logging.basicConfig(level=logging.INFO)
+
 # Dataset id
 DS_ID = 0
 
-DB_HOST = os.environ.get("DB_HOST")
-DB_PORT = os.environ.get("DB_PORT")
-DB_USER = os.environ.get("DB_USER")
-DB_PASSWORD = os.environ.get("DB_PASSWORD")
-DB_DB = os.environ.get("DB_DB")
-
+DB_URL = utilities.DB_URL
 DB_URL = "postgresql://{DB_USER}:{DB_PASSWORD}@{DB_HOST}:{DB_PORT}/{DB_DB}".format(
-    DB_HOST=DB_HOST,
-    DB_PORT=DB_PORT,
-    DB_USER=DB_USER,
-    DB_PASSWORD=DB_PASSWORD,
-    DB_DB=DB_DB,
+    DB_HOST="localhost",
+    DB_PORT=5433,
+    DB_USER="test",
+    DB_PASSWORD="example",
+    DB_DB="dataset",
```

Comment:
Maybe having this using the same approach as the other file would be better ?

Reply:
Shouldn't this come from the `utilities` you implemented in this PR ?